cli/storage_azure.go

```diff
@@ -0,0 +1,30 @@
+package cli
+
+import (
+	"context"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	"github.com/kopia/kopia/repo/blob"
+	"github.com/kopia/kopia/repo/blob/azure"
+)
+
+func init() {
+	var azOptions azure.Options
+
+	RegisterStorageConnectFlags(
+		"azure",
+		"an Azure blob storage",
+		func(cmd *kingpin.CmdClause) {
+			cmd.Flag("container", "Name of the Azure container").Required().StringVar(&azOptions.Container)
+			cmd.Flag("account-name", "Azure account name(overrides AZURE_ACCOUNT_NAME environment variable)").Required().Envar("AZURE_ACCOUNT_NAME").StringVar(&azOptions.AccountName)
```

Comment:
@PrasadG193  - The default Azure environment variables are `AZURE_STORAGE_ACCOUNT` and `AZURE_STORAGE_KEY`

We should use those names.

See https://docs.microsoft.com/en-us/azure/storage/common/storage-azure-cli

Reply:
@vkamra Looking at gcs tests, https://github.com/kastenhq/kopia/blob/kasten/repo/blob/gcs/gcs_storage_test.go#L21, they are appending `KOPIA_` to the env vars. So I followed the same pattern.

Reply:
Let me know that you think.

Reply:
I think it's ok to do that in the tests and we can stick with that pattern.

in this package - `cli` - we should use the az cli default environment variables - similar to https://github.com/kastenhq/kopia/blob/kasten/cli/storage_s3.go#L22

Reply:
Fixed!